cpmonitor/views.py

```diff
@@ -41,20 +50,30 @@ def _calculate_summary(request, node):
         subtasks = subtasks.filter(draft_mode=False)
 
     subtasks_count = len(subtasks)
-    status_counts = Counter([s.execution_status for s in subtasks])
+    status_counts = Counter(
+        [TO_STATUS[subtask.execution_status] for subtask in subtasks]
+    )
```

Comment:
I'd prefer a function instead of the constant dict here - but that's just taste, I guess.